### PR TITLE
Fix deploy command and improve build speed

### PR DESCRIPTION
**Pull request by meltsufin**
During refactoring I broke the "depoy" command by changing it to "cloud" accidentally. This fixes it.
Also, there is a performance issue with maven-jar-plugin when used with Java 7 that this fixes.

@loosebazooka  PTAL

### CODE REVIEW

File: src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java [hunk 1]
```diff
@@ -64,9 +66,20 @@ public int runDevAppServerCommand(List<String> args) throws ProcessRunnerExcepti
     command.add(getDevAppServerPath().toString());
     command.addAll(args);
 
+    sysOutCommand(command);
+
     return processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  private void sysOutCommand(List<String> command) {
+    StringBuilder commandString = new StringBuilder();
```
Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java]: Possible to use Guava Joiner here? http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/Joiner.html

Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java]: Also not sure I'd want this printed out by default. Would be related to log level wouldn't it?

Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java]: test should have caught this?

File: src/test/java/com/google/cloud/tools/app/CloudSdkAppEngineDeploymentTest.java [hunk 2]
```diff
@@ -78,7 +78,7 @@ public void testNewDeployAction_allFlags() throws AppEngineException, ProcessRun
     deployment.deploy(configuration);
 
     List<String> expectedCommand = ImmutableList
-        .of("cloud", appYaml1.toString(), "--bucket", "gs://a-bucket", "--docker-build",
+        .of("deploy", appYaml1.toString(), "--bucket", "gs://a-bucket", "--docker-build",
```
Comment (loosebazooka) [inline src/test/java/com/google/cloud/tools/app/CloudSdkAppEngineDeploymentTest.java]: oh I see why tests didn't catch this. lulz

File: src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java (hunk 1 above)
Comment (meltsufin) [inline src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java]: Joiner is a good idea. I was looking for something like that.

Comment (meltsufin) [inline src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java]: We're not passing a logger in at the moment. Maybe we should consider something like that.

**Comment by loosebazooka** [comment]
lgtm, (explore logging as future work?)

File: src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java [hunk 3]
```diff
@@ -64,9 +70,16 @@ public int runDevAppServerCommand(List<String> args) throws ProcessRunnerExcepti
     command.add(getDevAppServerPath().toString());
     command.addAll(args);
 
+    sysOutCommand(command);
+
     return processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  private void sysOutCommand(List<String> command) {
```
Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java]: This is just nitpicky, but this functions utility doesn't seem as obvious as before, feel like inline is easier to read?

**Comment by loosebazooka** [comment]
lgtm

**Pull request merged**